protos/metadata.proto

```diff
@@ -37,3 +37,29 @@ message TrafficMetadata {
   string user_agent = 6;
   string ping_health_endpoint = 7;
 }
+
+//Controller Health
+enum ControllerStatus {
```

Comment:
@dhruv0000  ControllerStatus, ControllerHealth, ControllerInfo, CommonError to meshkit. Just the service mesh metadata changes in here.